tests/INP-cylinder

```diff
@@ -1,9 +1,10 @@
 Cylinder test
-1 0 -1
+1 0 -1.1 -3
```

Comment:
doesn't this make an semi-infinite cylinder?

Do we still test the conventional RCC definition, or has this removed that?